### ISSUE TITLE
Power trigger combo bars work differently with 5.14.0

### ISSUE THREAD

**Issue by mrbuds** [WeakAuras/WeakAuras2#5175]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

It seems the maximum progress setting enabled with an empty value have a regression with power trigger

I don't understand from auras's data how the correct max progress was picked up

### WeakAuras Version

5.14.0

### World of Warcraft Flavor

Cataclysm Classic

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

I got this issue with only WeakAuras enabled

### Reproduction Steps

git checkout 5.13.2

=> import
```
!WA:2!TB5(ZTXvvC8SzBXnI2sImj06M246MeAlKeBLK6wgYdVYRJuWrsDLSZdciTA3R0UXR27M7Uso28mgO4cLxk8gkpeL3fkOYRcdpg)xWz0WVxW)g)cfYWm8RCU3DLLCACt70HzyyI(Hv3x79EUNZNZ3ZknBC)etsAeVwCZ4MlpYQLPmtctr3yotg1ZBRko2lUOoZC4cuQtGTxh96bwuwwVaBQRFm2czRuXNeK6pzn6)yxVyX)DBDxdC(CuB3GYjvZuqvRD4EMK6qzNusssorDMJ39yfe45)ooWbMxVkD)20dORwO68vMKm1bgJPBi2(hrZpqNfeRCfBxBFRyk4FbXwkGzxTkH5p0EzrnFIwMKY1RuPWcEewk1PZn1mtReGD00RZ0tSKVhXXjTPFSv9RxM0G4gKhxT9LAxm5e5lumFHj0kO5QxJ03kYXi4k0YNtD6P1eJuofr3jWsPoAfL9C0xGWAw3nYeIDv8gtyU6oZIMgA9xPPURDnD(f5rGBFQoMOLW7uKBxmFIb110hguXL6swHO7tYhWiUvdSgyjEp(vb2TsnDB3PGrGhage2nSNbG9I)Dhx7iSgHNPCl061ivfXgfhQU5SLd0DqBFhA1QJXVySfXJd9eBwb9jg7aE7X0mC0995Tu8TxKWB0PCu8pmMjllNtE09F4Y(06mdsz7AEuwGMVbUZ8jE0YHryRwH)RAI(JXgoN9LioTycRHFB0QYO19IwtE8OU1vcBhIqBPSpPgEp0gB)JU)rBfqnIUu32tdJyBQPrRwN0UcdJsONspqx(QHS2u8H4hq58j1uvZWUuet(h)Np5Z9VY8xoAeaM2fheHpBZ2juESSoZNZTGLPMUJNLUCzmCuXUASw8WIDiC322TcLffc9Ul7QUugjI8v5aHkJrz(LAJRYGigb7iULHwqPM(eNkICHwkzlui7PMwDQcXno3YhVLVNoBUtBBgyT9w24HMx4B35MGBjgCR782G3amqhXA0Obclqc2cedEJilfdUJyWDgt8Afn6gIKxL4Qx2HCcMUPnAkwqgyGLkRhL4L6y)9)a)3Vp1XEPxK)7VjlHzn(6bCUKyTI4WcT7Zk5ny1OTjlJ)uybTsLvl95YMPWetdp8SWBJJkW(2mSFrJdiEok)5Q(1O0aRCmAvgX33YBR19jtyEH6(betI5PSDTGhLXtnrZJfqUeAdism3co2UeLfP0AsAcZXcokg9HtidQTX4MDUQUvtP1a2SsgeJ9226CqNIAsuMyMczH30DUxyRlVTM4Pw0JopsMWDfRPOfp5BlDwBcE3sW2G4yY09YQ7kYZvqno6QI5lsXCv9akBOro6r8EZ83JrUyDBgjhFAo1zTeFuUMqj4EH7d2Em4E0eZQj2IHIF4L6M7hFm4UHHGDaVfyxX82gxMRbPqOac38F2nbdFLv9I8BHiXdVZnTz4(5sixRmWB9glmC1UzYPDNIrrOyv)aBJ5wyYidYcEqz4Ho5oGb7Gkh8fZXxxZObqWMeoWYd5fNhOk6BPBsN)mHGIutXy8hJShpVHxRxXWeNIEfdSRrkwJAwKtzLwnCp6MvaMEdkgH7X5jAgImcLjWUxnCFqretXGx2B3B027qQQBSqXkoukR0kIfWzn7klad0IXXdsbCqLmzZO2ju04SrxaVhAJ2uZfWcc2gfdSWyHf1XucgKH(e(s9UVn6Tc7j1smvVCZMrl1nOzIrhnXWk4(f5qxN9SZnTo38Aw59VrNNhJyylK(7iMDEuQ70mDVLoDuJoRDYIIJzNPW0PZO6T1(oBrOyL0zYOQvmSKDOpSbUXOGIL3DFDo9WgACJWS3zWL13tKlUlJeDwH3OEFfqJiz5I8QoYDI6e(QjqDbyWMCKumCuHIihk)TJ(qLU2yPUBgbR9aJ3D3OHwqzlIDvRGD1jsQjmXkX05p1QcfefScPjp)lXeto51kwPFPsqc4GqY)ANU1rePsLA2vBnbgILKBkKJ9TnjAAPprQc9lOEgP26r7iUFYTftLsyuh5Q9Y6wZ5ik3gTkBttIRwg1zv1u4hHf8oHJeU5fcVoEpyA(NEur3GC(K6i37wvrND(zsVVE92xE(lG1rxnjTwz6WIy(WpWH1MNNBnhCCzycqbgBTtpSa1AFaGmZ2LxpMyb2LAt7vuaolKEO9QzyrmMBkyRYJq9qPYLAOZS5vJaxTg6o1jXpmZWs3Tkb)Qn4fWQrV0X4)gsUxRLqDppUKmmdCY9ctxcExLGuXw(4qUTdp(6QmcAHvdFoliVW6GcIYDWmx36CWSwWPLGZG26RyDl4CwW7glnbNhEpW7f(zWphVSRvc6xeEs)qjWam7wJbi8AlqLTavxRocG(i4c8kgawbZPxzbOg4ou8db01vha8eY(WfFDP0dyro)EY5qa)r9LhcAibZJIZaIXlG29IW7dE)xg(aLGpi6V(qWhgUSeSeQTbFe4JkbFSWl5taFCyz0L)KsWNqg(KWtbFk4tdFg4ZcFolOjCf4ZVh4libFXWmz4ljdFz4RG7Xxf(AIHGVEj4PHXHVHe8n3f8TGVn0c(o9rxImQswW3Te89ezpW3h(bWMHYWpsg(XhP)pZbsEr4z79Ii8d)e4N(Yi5dj4xeJF(5GNPeuuc(Lwi9a)Qqaf(19Gt43aUWlGXHF7gYIWVRFiuT9jmxCHXhDbZd5FHE8ysPRLh73S)ViyIxU(4s12ANTwYaNtuo3fp3RF88G3epVH4zF84TiuH2iQ8GCCmsB9z4QzCQKRRSXy5bFvJLHcs)FN(yIBcGHay)QjBejM8PK5G61iq(8V8k9j48xejIKAekIkjBmjM41GaP9859YRpzk1XFS)NbivBtgx1qPMYJNE8zUoCjsEVcCjsTRNlh7MC5RbU82hqGz3yWCS(aZijKq2eX6nMnh7vnBwukEJ)8z(p
```
or https://wago.io/aETgwfDeF

Combo bars are empty or full as expected from combat bars, 

git checkout 5.14.0

now all bars progress ignore max progress setting

### Last Good Version

5.13.2

**Comment by mrbuds** [comment]
The max progress behavior change come from https://github.com/WeakAuras/WeakAuras2/commit/dbcb70b1e4df262af82f63620b3b0d80741e6df2

**Comment by InfusOnWoW** [comment]
Fixed in 27e221217ad0cfdc64debe308619fcd21ed1f771
